client/pkg/omnictl/kubeconfig.go

```diff
@@ -35,13 +35,22 @@ var kubeconfigCmdFlags struct {
 	cluster              string
 	forceContextName     string
 	serviceAccountUser   string
+	grantType            string
 	serviceAccountGroups []string
 	serviceAccountTTL    time.Duration
 	force                bool
 	merge                bool
 	serviceAccount       bool
 }
 
+var allGrantType = strings.Join([]string{
+	"auto",
+	"authcode",
+	"authcode-keyboard",
+	"password",
+	"device-code",
+}, "|")
```

Comment:
```suggestion
var allGrantTypes = strings.Join([]string{
	"auto",
	"authcode",
	"authcode-keyboard",
	"password",
	"device-code",
}, "|")
```

not all of these are working, right?

Reply:
Probably not, I haven't tried

Reply:
But the kubeconfig is valid with that, so I don't know

Reply:
Should we validate against only the ones we know to be working? To avoid generating broken kubeconfigs?

Reply:
I guess that makes sense

Reply:
updated. Left only `auto`, `authcode` and `authcode-keyboard`. Other 2 don't work